docs/.vitepress/vitepress/styles/common/helper.scss

```diff
@@ -15,4 +15,10 @@
       linear-gradient(45deg, #0000 75%, #292929 75%),
       linear-gradient(135deg, #0000 75%, #292929 75%);
   }
+  ::-webkit-scrollbar-thumb {
+    background-color: rgba(255, 255, 255, 0.2);
+    &:hover {
+      background-color: rgba(255, 255, 255, 0.4);
+    }
+  }
```

Comment:
It feel more appropriate to move to the `scrollbar.scss` file.

Reply:
reslove.